src/js/page_sections/Menu/MenuItem.js

```diff
@@ -40,6 +40,7 @@ const MenuItem = ({theme, id, openSection, handleSublistToggle, handleMenuToggle
           openSection={openSection}
           theme={theme}
           handleMenuToggle={handleMenuToggle}
+          intl={intl}
```

Comment:
same here, can we inject intl where needed via wrapping the component with injectIntl like in the above comment.